package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Last time I've was added global package.json, it caused cache issues within rush build's, not sure it is a good idea to add it.